src/styles/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  padding: 10px;
+  display: flex;
+  gap: $gap;
```

Comment:
The variable `$gap` is used here but is not defined within this file. Ensure that `$gap` is defined either in this file or imported from another SCSS file.